### ISSUE TITLE
Crowd seeker: comfirm at the same time?

### ISSUE THREAD

**Issue by Mjnstag** [McTsts/Werewolves-Roles#62]
I remember there being some debate/discussion over cs only using 1 guess at a time and basing their remaining guesses on the results. Do we want this/clarify it that guesses can(not) be confirmed separately?

**Comment by McTsts** [comment]
I think I'd prefer being allowed to do them separately, but I'm not sure. With the current role description it could probably be either way, it's pretty unclear.

**Comment by venomousbirds** [comment]
Allowing them to do it seperately could mean they check the same person 3 times, which seems kind of weird to me. I prefer having all checks done at the same time, both for convenience and to avoid people checking the same person

**Comment by McTsts** [comment]
"The Crowd Seeker can select up to three different players each night" - they cannot select the same player several times

**Comment by Mjnstag** [comment]
i mean, it says "up to" so can also be less?

**Comment by McTsts** [comment]
Yes, but from my understanding there's nothing that suggests you could select the same player several times.
> The Crowd Seeker can select up to three different players each night, and guess for each of them what their role could be. 
The guessing and selecting is connected, and the selecting requires different players. So you can select 0 or 1 player or 2 or 3 different players.
All of this should probably be clarified, but in my opinion the current role description makes the most sense to be interpreted as:

- Select 3 or less players, but each player only once  ("three different players")
- Can be split into 3 different actions ("guess for each")

**Comment by venomousbirds** [comment]
I think it won't do any harm if we clarify that all checks need to be made at the same time in that case. It's been used to check claims which talking in a cc, and I don't think the role should be used like that.

**Comment by McTsts** [comment]
Well, if they are all at the same time the role is a bit weaker because sometimes you can skip some checks if the first one is successful (e.g. if several people claim the same role but it only exists once it could be done with a single check if you select the right person first)

**Comment by MrTurtleJr** [comment]
I personally prefer the whole "you need to check at the same time", even if it weakens the CS, as that was the way I interpreted the role and I just thinks it makes more sense.
